test/hpack/huffman_test.exs

```diff
@@ -11,16 +11,31 @@ defmodule HuffmanTest do
 
   test "decode a sentence" do
     hello_world = <<
-      0x27::6, 0x5::5, 0x28::6, 0x28::6, 0x7::5, 0x14::6,
-      0x78::7, 0x7::5, 0x2c::6, 0x28::6, 0x24::6, 0x3f8::10
+      0x27::6,
+      0x5::5,
+      0x28::6,
+      0x28::6,
+      0x7::5,
+      0x14::6,
+      0x78::7,
+      0x7::5,
+      0x2C::6,
+      0x28::6,
+      0x24::6,
+      0x3F8::10
```

Comment:
Please revert these reformattings. I know this is what the formatter does but the blocks match the RFCs and also keep the specs readable. At least I can not read hpack encoded binary on the fly.

Reply:
Done.